echemdb/entry.py

```diff
@@ -49,23 +51,21 @@
 import logging
 import os.path
 
-from echemdb.cv.descriptor import Descriptor
+from echemdb.descriptor import Descriptor
 
 logger = logging.getLogger("echemdb")
 
 
 class Entry:
     r"""
     A [data packages](https://github.com/frictionlessdata/datapackage-py)
```

Comment:
This link does not render correctly https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks